contrib/scalar/scalar.c

```diff
@@ -553,24 +713,22 @@ static int cmd_clone(int argc, const char **argv)
 	if ((res = run_git(NULL, "-c", buf.buf, "init", "--", dir, NULL)))
 		goto cleanup;
 
-	/* TODO: trace command-line options, is_unattended, elevated, dir */
+	/* common-main already logs `argv` */
+	trace2_data_string("scalar", the_repository, "dir", dir);
 	trace2_data_intmax("scalar", the_repository, "unattended",
 			   is_unattended());
 
-	/*
-	 * TODO: verify that the file system is case-insensitive on Windows and
-	 * macOS, and case-sensitive on Linux.
-	 */
-
-	if ((res = set_acls(dir)) < 0)
-		goto cleanup;
-
 	if (!branch &&
 	    !(branch = remote_default_branch(dir, url))) {
 		res = error(_("failed to get default branch for '%s'"), url);
 		goto cleanup;
 	}
 
+	/*
+	 * TODO: verify that the file system is case-insensitive on Windows and
+	 * macOS, and case-sensitive on Linux.
+	 */
+
```

Comment:
This shouldn't be important anymore. That was mostly for VFS for Git things that cared about that. We are less coupled to caring about case sensitivity now.

Reply:
Excellent! I'll drop it, then.

Reply:
Done. I also switched things around so that the commit history I actually want to keep is _not_ based on the current `features/scalar` branch, but rather on the commit where it branched off of `vfs-2.31.1`.